src/pytest_inmanta_lsm/failed_resources_logs.py

```diff
@@ -34,9 +34,8 @@ def _extract_logs(self, get_version_result: Dict[str, Any]) -> List[Tuple[str, s
         for resource in get_version_result["resources"]:
             resource_id = resource["resource_id"]
 
-            # Only interested in failed resources
-            if resource["status"] != "failed":
-                continue
+            if resource["status"] in ["deploying", "skipped", "failed"]:
```

Comment:
I'm not very familiar with this part of the project, but it might be better for maintainability to use an Enum ?

Reply:
in general yes
This enum exist in core

I did it like this because it was like this.